opensrp-immunization/src/test/resources/robolectric.properties

```diff
@@ -1 +1 @@
-sdk=28
+sdk=27
```

Comment:
Curious to learn why the `sdk` version is being lowered

Reply:
In some test Classes 27 was hardcoded, updated globally for consistency. Have since reverted since it works with 28 locally.